tests/foreman/ui/test_rhc.py

```diff
@@ -44,9 +44,17 @@ def fixture_enable_rhc_repos(module_target_sat):
 @pytest.fixture(scope='module')
 def module_rhc_org(module_target_sat):
     """Module level fixture for creating organization"""
-    org = module_target_sat.api.Organization(
-        name=settings.rh_cloud.organization or gen_string('alpha')
-    ).create()
+    if settings.rh_cloud.crc_env == 'prod':
+        org = module_target_sat.api.Organization(
+            name=settings.rh_cloud.organization or gen_string('alpha')
+        ).create()
+    else:
+        org = (
+            module_target_sat.api.Organization()
+            .search(query={'search': f'name="{settings.rh_cloud.organization}"'})[0]
```

Comment:
There is no value set for this org setting in rhcloud yaml, I wonder how this search works for you? Could you provide some test results?